### PR TITLE
Added definitions for bytebuffer.js and long.js

### PR DESCRIPTION
**Pull request by dcappellin**

### CODE REVIEW

**Comment by vvakame** [comment]
please move bytebuffer/long.d.ts to long/long.d.ts

**Comment by cappellin** [comment]
Moved long.js files to folder long and updated the version of long.js in that folder with mine that is newer. Thanks

File: long/long.d.ts [hunk 1]
```diff
@@ -1,75 +1,352 @@
-// Type definitions for Long.js v2.2.5
-// Project: https://github.com/dcodeIO/Long.js
-// Definitions by: Peter Kooijmans <https://github.com/peterkooijmans/>
```
Comment (vvakame) [inline long/long.d.ts]: please keep original author name.

**Comment by vvakame** [comment]
@peterkooijmans could you review this PR?
@cappellin bytebuffer/long.d.ts is still alive. is it ok?

**Comment by cappellin** [comment]
Removed long.js from bytebuffer folder.

**Comment by cappellin** [comment]
Fixed the signature of concat.

**Comment by vvakame** [comment]
*long/long.d.ts*

to author (@peterkooijmans). Could you review this PR?
:+1: or :-1:?

Checklist

* [ ] pass the Travis CI test?

**Comment by cappellin** [comment]
@vvakame shall we ask to @dcodeIO for PR?

**Comment by cappellin** [comment]
Ops... my fault I wrongly added a new definition - for pako - to my master, so it's visible here. I apologize. What should I do?

**Comment by vvakame** [comment]
*pako/pako.d.ts*

Checklist

* [ ] is correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/pako
  * http://bower.io/search/?q=pako
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (pako/pako-tests.ts or pako/pako-tests.tsx)
* [ ] pass the Travis CI test?

**Comment by vvakame** [comment]
okay, i decided to merge this PR. thanks mate.

**Pull request merged**